AzureResourceManager/Artifactory/MP_submission/mainTemplate.json

```diff
@@ -702,7 +703,7 @@
       }
     },
     {
-      "apiVersion": "2019-06-01",
+      "apiVersion": "2020-08-01",
```

Comment:
Check the API version here - https://docs.microsoft.com/en-us/azure/templates/microsoft.storage/2021-02-01/storageaccounts?tabs=json

Reply:
Updated apiVersion from this documents.